File: 1344_Angle_Between_Hands_of_a_Clock/solution.c
double angleClock(int hour, int minutes) {
    double h = (hour % 12) * 30.0 + minutes * 0.5, m = minutes * 6.0;
    double d = fabs(h - m);
    return d < (360.0 - d) ? d : (360.0 - d);
}

File: 1344_Angle_Between_Hands_of_a_Clock/solution.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        h, m = (hour % 12) * 30.0 + minutes * 0.5, minutes * 6.0
        d = abs(h - m)
        return min(d, (360.0 - d))
